Dockerfile

```diff
@@ -9,6 +9,9 @@ RUN dnf install -y \
     yarnpkg \
     maven \
     rubygem-bundler \
+    ruby-devel \
```

Comment:
Why were ruby-devel and gcc-c-++ necessary? We're not currently including them in deptopia builds

Reply:
I found that that bundler command used by deplist didn't work in the container without these dependencies. I think they are required to compile native extensions in 'unf' gem.

Reply:
I seems it's actually the unf-ext gem.